src/webui/src/static/function.ts

```diff
@@ -43,7 +43,7 @@ const getFinalResult = (final?: MetricDataRecord[]): number => {
     let acc;
     let showDefault = 0;
     if (final) {
-        acc = JSON.parse(final[final.length - 1].data);
+        acc = JSON5.parse(final[final.length - 1].data);
```

Comment:
Is it possible to call parseMetrics here?  there maybe more updates in parseMetrics later.

Reply:
It's OK.